test/TritonGPU/accelerate-matmul.mlir

```diff
@@ -93,3 +93,31 @@ module attributes {"triton_gpu.compute-capability" = 80 : i32, "triton_gpu.num-c
     tt.return %d : tensor<64x64xf32, #blocked>
   }
 }
+
+// -----
+
+// CHECK: #mma = #triton_gpu.nvidia_mma<{versionMajor = 3, {{.*}}, instrShape = [16, 32, 16]}>
+#blocked = #triton_gpu.blocked<{sizePerThread = [4, 4], threadsPerWarp = [1, 32], warpsPerCTA = [32, 1], order = [1, 0], CTAsPerCGA = [1, 1], CTASplitNum = [1, 1], CTAOrder = [1, 0]}>
+module attributes {"triton_gpu.compute-capability" = 90 : i32, "triton_gpu.num-ctas" = 1 : i32, "triton_gpu.num-warps" = 32 : i32, "triton_gpu.threads-per-warp" = 32 : i32} {
+  tt.func @check_instrShape_per_warps(%arg0: !tt.ptr<i8, 1> {tt.divisibility = 16 : i32}, %arg1: !tt.ptr<f16, 1> {tt.divisibility = 16 : i32}, %arg2: !tt.ptr<f16, 1> {tt.divisibility = 16 : i32}) {
+    // CHECK-LABEL: check_instrShape_per_warps
+    %a_ptr = tt.splat %arg0 : (!tt.ptr<i8, 1>) -> tensor<128x128x!tt.ptr<i8, 1>, #blocked>
+    %mask = arith.constant dense<true> : tensor<128x128xi1, #blocked>
+    %zero_i8 = arith.constant dense<0> : tensor<128x128xi8, #blocked>
+    %a0 = tt.load %a_ptr, %mask, %zero_i8 {boundaryCheck = array<i32: 0, 1>, cache = 1 : i32, evict = 1 : i32, isVolatile = false, padding = 1 : i32} : tensor<128x128xi8, #blocked>
+    %a1 = arith.sitofp %a0 : tensor<128x128xi8, #blocked> to tensor<128x128xf16, #blocked>
+    %a2 = triton_gpu.convert_layout %a1 : (tensor<128x128xf16, #blocked>) -> tensor<128x128xf16, #triton_gpu.dot_op<{opIdx = 0, parent = #blocked}>>
+
+    %b_ptr = tt.splat %arg1 : (!tt.ptr<f16, 1>) -> tensor<128x128x!tt.ptr<f16, 1>, #blocked>
+    %zero_f16 = arith.constant dense<0.000000e+00> : tensor<128x128xf16, #blocked>
+    %b0 = tt.load %b_ptr, %mask, %zero_f16 {boundaryCheck = array<i32: 0, 1>, cache = 1 : i32, evict = 1 : i32, isVolatile = false, padding = 1 : i32} : tensor<128x128xf16, #blocked>
+    %b1 = triton_gpu.convert_layout %b0 : (tensor<128x128xf16, #blocked>) -> tensor<128x128xf16, #triton_gpu.dot_op<{opIdx = 1, parent = #blocked}>>
+
+    %zero_f32 = arith.constant dense<0.000000e+00> : tensor<128x128xf32, #blocked>
+    %result = tt.dot %a2, %b1, %zero_f32 {allowTF32 = true, maxNumImpreciseAcc = 0 : i32} : tensor<128x128xf16, #triton_gpu.dot_op<{opIdx = 0, parent = #blocked}>> * tensor<128x128xf16, #triton_gpu.dot_op<{opIdx = 1, parent = #blocked}>> -> tensor<128x128xf32, #blocked>
+    %result1 = arith.truncf %result : tensor<128x128xf32, #blocked> to tensor<128x128xf16, #blocked>
+    %result_ptr = tt.splat %arg2 : (!tt.ptr<f16, 1>) -> tensor<128x128x!tt.ptr<f16, 1>, #blocked>
+    tt.store %result_ptr, %result1, %mask {cache = 1 : i32, evict = 1 : i32} : tensor<128x128xf16, #blocked>
+    tt.return
+  }
+}
```

Comment:
nit: Please you minimize the test. (I believe you only need a tt.dot and maybe a tt.store here).